### ISSUE TITLE
Julia 0.7 test XLSX error

### ISSUE THREAD

**Issue by programistawpf** [felipenoris/XLSX.jl#59]
Will XLSX work on Julia 0.7 ? 

```
test XLSX
......
┌ Warning: `method_exists(f, t)` is deprecated, use `hasmethod(f, t)` instead.
│   caller = top-level scope at none:0
└ @ Core none:0
WARNING: importing deprecated binding Base.Dates into XLSX.
WARNING: Base.Dates is deprecated, run `using Dates` instead
  likely near C:\Users\PC\.julia\packages\XLSX\ETIGH\src\structs.jl:45
WARNING: Base.Dates is deprecated, run `using Dates` instead
  likely near C:\Users\PC\.julia\packages\XLSX\ETIGH\src\structs.jl:45
WARNING: Base.Dates is deprecated, run `using Dates` instead
  likely near C:\Users\PC\.julia\packages\XLSX\ETIGH\src\structs.jl:45
WARNING: importing deprecated binding Base.Nullable into XLSX.
ERROR: LoadError: LoadError: TypeError: in Type{...} expression, expected UnionAll, got typeof(Base.Nullable)
Stacktrace:
 [1] top-level scope at none:0
 [2] include at .\boot.jl:317 [inlined]
 [3] include_relative(::Module, ::String) at .\loading.jl:1038
 [4] include at .\sysimg.jl:29 [inlined]
 [5] include(::String) at C:\Users\PC\.julia\packages\XLSX\ETIGH\src\XLSX.jl:3
 [6] top-level scope at none:0
 [7] include at .\boot.jl:317 [inlined]
 [8] include_relative(::Module, ::String) at .\loading.jl:1038
 [9] include(::Module, ::String) at .\sysimg.jl:29
 [10] top-level scope at none:2
 [11] eval at .\boot.jl:319 [inlined]
 [12] eval(::Expr) at .\client.jl:399
 [13] top-level scope at .\none:3
in expression starting at C:\Users\PC\.julia\packages\XLSX\ETIGH\src\structs.jl:153
in expression starting at C:\Users\PC\.julia\packages\XLSX\ETIGH\src\XLSX.jl:14
ERROR: LoadError: Failed to precompile XLSX [fdbf4ff8-1666-58a4-91e7-1b58723a45e0] to C:\Users\PC\.julia\compil
d\v0.7\XLSX\gPxqz.ji.
Stacktrace:
 [1] error(::String) at .\error.jl:33
 [2] macro expansion at .\logging.jl:313 [inlined]
 [3] compilecache(::Base.PkgId, ::String) at .\loading.jl:1185
 [4] _require(::Base.PkgId) at .\logging.jl:311
 [5] require(::Base.PkgId) at .\loading.jl:852
 [6] macro expansion at .\logging.jl:311 [inlined]
 [7] require(::Module, ::Symbol) at .\loading.jl:834
 [8] include at .\boot.jl:317 [inlined]
 [9] include_relative(::Module, ::String) at .\loading.jl:1038
 [10] include(::Module, ::String) at .\sysimg.jl:29
 [11] include(::String) at .\client.jl:398
 [12] top-level scope at none:0
in expression starting at C:\Users\PC\.julia\packages\XLSX\ETIGH\test\runtests.jl:2
ERROR: Package XLSX errored during testing
```
Paul

**Comment by felipenoris** [comment]
Yes. This package is tested for Julia v0.7, v1.0 and nighly for windows, OSX and Linux.

Looks like you don't have the latest version for this package. Try running `Pkg.update()`.

Can you post the result of `Pkg.status()` here?

**Comment by programistawpf** [comment]
Now is workin , bu very slow !
  | | |_| | | | (_| |  |  Version 0.7.0 (2018-08-08 06:46 UTC)
 _/ |\__'_|_|_|\__'_|  |  Official http://julialang.org/ release
|__/                   |  x86_64-w64-mingw32

(v0.7) pkg> up
  Updating registry at `C:\Users\PC\.julia\registries\General`
  Updating git-repo `https://github.com/JuliaRegistries/General.git`
 Resolving package versions...
 Installed EzXML ─── v0.9.0
 Installed LibCURL ─ v0.4.1
  Updating `C:\Users\PC\.julia\environments\v0.7\Project.toml`
 [no changes]
  Updating `C:\Users\PC\.julia\environments\v0.7\Manifest.toml`
  [8f5d6c58] ↑ EzXML v0.8.0 ? v0.9.0
  [b27032c2] ↑ LibCURL v0.4.0 ? v0.4.1
  Building EzXML ──→ `C:\Users\PC\.julia\packages\EzXML\DUxj7\deps\build.log`
  Building LibCURL → `C:\Users\PC\.julia\packages\LibCURL\OoXMv\deps\build.log`

(v0.7) pkg> status
    Status `C:\Users\PC\.julia\environments\v0.7\Project.toml`
  [89b67f3b] ExcelFiles v0.6.1
  [c04bee98] ExcelReaders v0.10.2
  [5789e2e9] FileIO v1.0.1
  [28b8d3ca] GR v0.34.1
  [f67ccb44] HDF5 v0.10.2
  [4138dd39] JLD v0.8.3
  [033835bb] JLD2 v0.1.2
  [429524aa] Optim v0.16.0
  [91a5bcdd] Plots v0.19.3
  [69024149] StringEncodings v0.3.1
  [fdbf4ff8] XLSX v0.4.1

(v0.7) pkg>`

`
julia> @time XLSX.readtable("dane\\dane.xlsx","Arkusz1")
 78.841218 seconds (470.19 M allocations: 14.590 GiB, 3.00% gc time)
(Any[Any[2015-01-01, 2015-02-01, 2015-03-01, 2015-04-01, 2015-05-01, 2015-`

table size is 47 x 1500 reding very  long time :/

**Comment by felipenoris** [comment]
I just ran a benchmark on a file with the same dimensions, and it took 5 seconds the file on a mac.
There are many factors that could slow you down though. I could take a look if it is ok for you to send me the file you're trying to read.

**Comment by programistawpf** [comment]
Which XLSX script do you load the all sheet into the table in the 
fastest and easiest way? Paul


W dniu 2018-09-14 o 15:47, Felipe Noronha pisze:
> I just ran a benchmark on a file with the same dimensions, and it took 
> 5 seconds the file on a mac.
> There are many factors that could slow you down though. I could take a 
> look if it is ok for you to send me the file you're trying to read.

**Comment by felipenoris** [comment]
Vc é brasileiro?

**Comment by programistawpf** [comment]
No, Turk ;)
W dniu 2018-09-14 o 16:05, Felipe Noronha pisze:
> Vc é brasileiro?

**Comment by felipenoris** [comment]
Oh.. Well. I didn't understand what you meant by "Which XLSX script do you load the all sheet into the table in the fastest and easiest way?" . Can you rephrase that?

**Comment by programistawpf** [comment]
Now i must read data in 2 step:
`
julia> xf=XLSX.readxlsx("dane\\dane.xlsx")
XLSXFile("dane.xlsx") containing 1 Worksheets
             sheetname size          range
-------------------------------------------------
               Arkusz1 37x1104       A1:APL37


julia> xf["Arkusz1"][:]
37×1104 Array{Any,2}:
`

How to read to Array Arkusz1 in 1 step is possibile it ?
Paul


W dniu 2018-09-14 o 17:13, Felipe Noronha pisze:

**Comment by felipenoris** [comment]
Yes. If you have tabular data, you can use `readtable`. If you need a customized cell range, you can use `readdata`.

**Comment by felipenoris** [comment]
I'm closing this since this is not a bug.
